spec/models/ext_management_system_spec.rb

```diff
@@ -69,6 +69,7 @@
       "ibm_cloud_power_virtual_servers"         => "IBM Power Systems Virtual Servers",
       "ibm_cloud_power_virtual_servers_storage" => "IBM Power Systems Virtual Servers Storage",
       "ibm_terraform_configuration"             => "IBM Terraform Configuration",
+      "ibm_vpc"                                 => "IBM Virtual Private Cloud",
```

Comment:
@agrare I'm surprised this isn't `ibm_cloud_vpc` (or perhaps `ibm_cloud_power_virtual_servers` should be `ibm_power_virtual_servers`)

Reply:
Also is this in #19440 ?

Reply:
Yeah ibm_cloud_vpc would be more consistent and yes we have this tracked there in `ExtManagementSystem spec has a test that expects to know every single plugin`